front/app/translations/admin/ar-MA.json

```diff
@@ -13,7 +13,6 @@
   "app.components.ConfirmationModal.disabled": "تم التعطيل",
   "app.components.ConfirmationModal.enabled": "تم التمكين",
   "app.components.ConfirmationModal.whenTurnedOnUsersWillHaveToConfirm": "عند تشغيل هذه الميزة، سيتوجب على المستخدمين الجدد تأكيد عناوين بريدهم الإلكتروني بإدخال رمز من 4 أرقام يُرسل إليهم عبر البريد الإلكتروني لإتمام عملية التسجيل.",
-  "app.components.ProjectTemplatePreview.citizenlabExpert": "خبير CitizenLab",
```

Comment:
But you've changed the translation key, so why would you need these back?

Reply:
I think otherwise the translations would get lost in Crowdin and translators would have to translate them again. But the script replaces the translation key and so we would get the desired translation files if we run the script after reverting (without any effort needed from translators).

Reply:
> These were automatically deleted, but I propose to revert the commit https://github.com/CitizenLabDotCo/citizenlab/commit/1bd388b445c992477d974aeadebf3dc04a6a8d19 just before generating the final translations (which we should do just before releasing this).

@sebastienhoorens I think it was correct that these were deleted? :thinking: We've replaced them with new strings using a "Go Vocal" identifier right? So we expect the old ones to be deleted here?

Reply:
@amanda-anderson  Yes, it's normal that they got deleted, but if we restore them only on a local branch (not merging the revert to master or production), then we should hopefully get the right translations without any work needed from translators.